### PR TITLE
Documentation of kafka-ws intercommunication

### PR DESCRIPTION
**Pull request by thiagodpf**
This documentation aims to detail the interaction between the components from obtaining the ticket to establishing the communication via websocket.

I didn't go into more detail about the kafka-ws component, I believe that this is still in progress by the team.

### CODE REVIEW

File: source/locale/pt_BR/LC_MESSAGES/internal-communication.po [hunk 1]
```diff
@@ -633,16 +634,104 @@ msgid ""
...
+#: ../../source/internal-communication.rst:582
+msgid ""
+"Since the ticket is valid, that is, it corresponds to an entry on Redis, "
+"Kafka-ws retrieves the payload related to the ticket, verifies the "
+"integrity of the payload and deletes that entry on Redis so that the "
+"ticket cannot be used again."
+msgstr ""
+"Dado que o ticket esteja válido ou seja, corresponde a uma entrada no Redis, "
+"o Kafka-ws recupera o payload relacionado ao ticket, verifica a integridade "
+"do payload e exclui essa entrada no Redis para que o ticket não possa ser "
```
Comment (eduardogmisiuk) [inline source/locale/pt_BR/LC_MESSAGES/internal-communication.po:719]: verifica sua integridade e exclui

File: source/internal-communication.rst [hunk 2]
```diff
@@ -552,12 +552,92 @@ will sign a certificate and link this certificate to the device registration.
 The ``x509-identity-mgmt`` component is responsible for providing
 certificate-related services for devices.
 
+Kafka-ws
+---------------------
+
+Kafka WebSocket service allows the users to retrieve data from a given dojot
+topic in a Kafka cluster, this retrieval can be conditional and/or partial.
```
[the remark this answers is not in the document]
Comment (thiagodpf) [reply source/internal-communication.rst:559]: Great!

File: source/locale/pt_BR/LC_MESSAGES/internal-communication.po [hunk 3]
```diff
@@ -646,92 +645,96 @@ msgstr ""
...
+"para websocket. Se a autorização for concedida, o Kafka-WS abre um canal "
 "de subscrição com base em um tópico específico no Kafka. A partir daí, o "
-"upgrade para websocket é estabelecido e o usuário começa a receber os dados "
-"à medida que vão sendo publicados no Kafka."
+"upgrade para websocket é estabelecido e o usuário começa a receber os "
+"dados à medida que vão sendo publicados no Kafka."
+
+#: ../../source/internal-communication.rst:593
+msgid "Obtaining a ticket and connecting via websocket"
+msgstr ""
```
Comment (eduardogmisiuk) [inline source/locale/pt_BR/LC_MESSAGES/internal-communication.po:737]: Isn't it missing a translation?

Comment (thiagodpf) [reply source/locale/pt_BR/LC_MESSAGES/internal-communication.po:737]: yes!

**Pull request merged**